app/forms/common/YesNoAnswer.scala

```diff
@@ -35,28 +33,15 @@ case class YesNoAnswer(answer: String) extends Ordered[YesNoAnswer] with Amendme
   override def compare(that: YesNoAnswer): Int = answer.compare(that.answer)
 
   def value: String = answer
-
-  def valueAdded(pointer: ExportsFieldPointer)(implicit messages: Messages): String =
-    forAddedValue(pointer, messages(mappingsForAmendment(pointer)), valueForYesNo(answer))
-
-  def valueAmended(newValue: Amendment, pointer: ExportsFieldPointer)(implicit messages: Messages): String =
-    forAmendedValue(pointer, messages(mappingsForAmendment(pointer)), valueForYesNo(answer), valueForYesNo(newValue.value))
-
-  def valueRemoved(pointer: ExportsFieldPointer)(implicit messages: Messages): String =
-    forRemovedValue(pointer, messages(mappingsForAmendment(pointer)), valueForYesNo(answer))
 }
 
 object YesNoAnswer {
 
   implicit val format: OFormat[YesNoAnswer] = Json.format[YesNoAnswer]
 
   private lazy val parties = s"${ExportsDeclaration.pointer}.${Parties.pointer}"
-  private lazy val transport = s"${ExportsDeclaration.pointer}.${Transport.pointer}"
 
-  lazy val mappingsForAmendment = Map(
-    s"$parties.personPresentingGoodsDetails.eori" -> "declaration.summary.parties.eidr",
-    s"$transport.expressConsignment" -> "declaration.summary.transport.expressConsignment"
```

Comment:
Just to be sure... don't we need any more the `expressConsignment` mapping?

Reply:
We don't need this construct at all now. Will remove it from the code completely :+1: